src/components/user-profile.html

```diff
@@ -136,9 +171,9 @@ <h3 class="div-center" style="margin: initial;padding-bottom: inherit;">User Pro
 
       expandCollapseCard() {
         this.profileExpanded ? this.profileExpanded = false :
```

Comment:
I think this is what you're going for, and much simpler:
```suggestion
        this.profileExpanded = !this.profileExpanded;
```